Ldtpd/Value.cs

```diff
@@ -323,7 +327,7 @@ public int Increase(string windowName, string objName, int iterations)
         }
         public int Decrease(string windowName, string objName, int iterations)
         {
-            double min = GetMinValue(windowName, objName);
+            double min = Double.MinValue;//GetMinValue(windowName, objName);
```

Comment:
Why changing this ? We need to get the min value of current object, rather than the actual double min.